ci_support/run_docker_build.sh

```diff
@@ -37,70 +39,50 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
-# library dependencies for matplotlib
-yum install -y libXext libXrender libSM tk libX11-devel
```

Comment:
We still need this for the conda build of arm_pyart.  Holding off merging this until these packages make it into the Docker image